rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/FileDownloadRequestChain.java

```diff
@@ -109,7 +110,7 @@ public Long call() throws IOException
     long startTime = System.currentTimeMillis();
     File file = new File(localFile);
     if (!file.exists()) {
-      throw new IOException(String.format("File does not exists %s", localFile));
+      throw new FileNotFoundException(String.format("File does not exists %s", localFile));
```

Comment:
nit: Make the exception class changes in separate commit

Reply:
done